owlbot.py

```diff
@@ -30,6 +23,3 @@
   '.kokoro/publish.sh',
   '.kokoro/release/publish.cfg'
 ])
-subprocess.run(['npm', 'install'])
-subprocess.run(['npm', 'run', 'update-disclaimers'])
```

Comment:
We actually need these :) This is what runs the generator nightly, and submits PRs for us.

Reply:
Ok, then Owl Bot cannot do the job.  We need to find another tool.